e2e/openssl/openssl_test.go

```diff
@@ -120,6 +120,22 @@ func TestOpenSSL(t *testing.T) {
 		require.NoError(err, "stderr: %q", stderr)
 	})
 
+	t.Run("certificates pass strict verification", func(t *testing.T) {
+		require := require.New(t)
+
+		ctx, cancel := context.WithTimeout(context.Background(), 1*time.Minute)
+		defer cancel()
+
+		c := kubeclient.NewForTest(t)
+
+		require.NoError(c.WaitFor(ctx, kubeclient.Deployment{}, ct.Namespace, opensslBackend))
+
+		cmd := `openssl verify -x509_strict -CAfile /tls-config/coordinator-root-ca.pem -untrusted /tls-config/certChain.pem /tls-config/certChain.pem`
```

Comment:
Could you add a comment similar to the one on the previous test case explaining what exactly we are checking here (and why it isn't covered by the previous commend)? I always find these openssl commands hard to understand.

Reply:
Thanks for the suggestion - after reading through the manpages again, I realized that I can add `x509_strict` to the `s_client` command, which should then be equivalent to the checks I introduced here. Removing this test case again and adding the option to the `opensslConnectCmd`.